src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1352,10 +1351,13 @@ void runtime::cpu::CPU_ExternalFunction::build()
             };
         }
 
-        enables.emplace_back(make_pair(enable, functors.size() - functor_count));
+        enables.emplace_back(make_pair(enable, 1));
```

Comment:
I was more implying that we could remove it altogether (or use some other container) since it's not very useful to store a '1' there, right?